lambdas/service/app.py

```diff
@@ -394,16 +394,17 @@ def get_pagination(self, entity_type: str) -> Pagination:
     # FIXME: Return furl instance
     #        https://github.com/DataBiosphere/azul/issues/3398
```

Comment:
These FIXMEs should be removed as we are addressing the issue in this PR, please confirm all related FIXMEs are removed from the codebase.